### PR TITLE
Add microinteration to product switcher

### PR DESCRIPTION
**Pull request by PascalSenn**

### CODE REVIEW

File: website/src/components/misc/link.tsx [hunk 1]
```diff
@@ -17,12 +14,18 @@ export const Link: FunctionComponent<GatsbyLinkProps<unknown>> = ({
       className={className}
       activeClassName={activeClassName}
       partiallyActive={partiallyActive}
+      ref={ref as string & React.RefObject<HTMLAnchorElement>}
```
Comment (PascalSenn) [inline website/src/components/misc/link.tsx:17]: @rstaib  only got it working like this..

File: website/src/components/misc/winking-logo.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import { motion, SVGMotionProps } from "framer-motion";
+
+const transition = {
+  duration: 1.1,
+  times: [0, 0.35, 0.6, 0.8, 0.9, 1],
+};
+
+const glassesStages = [
+  "m 612.43331,842.49922 c 0,0 61.66627,-7.5 72.36627,-7.5 0.052,0.47569 0.1524,0.94481 0.3,1.4 10.6,33.6 10.4,71.1 26.8,102.8 6.5,12.5 15.3,26.9 28.3,33.8 30,16 163.2,19.8 192.7,4.7 43.3,-22.2 75.20002,-138.2 75.40002,-141.7 8.3,0 77.3,0.1 85.6,0 0.2,3.5 32.1,119.5 75.4,141.7 29.5,15.2 162.7,11.3 192.7,-4.7 13,-6.9 21.8,-21.3 28.3,-33.8 16.4,-31.7 16.2,-69.2 26.8,-102.8 0.1476,-0.45519 0.2481,-0.92431 0.3,-1.4 12.4,0.1 61.1363,7.08726 75.8363,9.18726 0,0 -1.485,-67.6816 -3.8605,-69.74787 -2.6797,-2.33083 -190.7667,-4.6886 -200.4317,-5.44994 -6.7531,-0.53196 -173.1086,1.83851 -204.4058,2.88867 -23.1742,0.7776 -56.8563,0.93105 -64.7366,0.78641 -8.3223,-0.15275 -138.07252,-4.98422 -207.58982,-4.33973 -48.731,0.45178 -185.3393,6.29683 -185.3393,6.29683 -1.2285,2.80403 -12.24427,17.43256 -14.43887,67.87837 z",
```
Comment (rstaib) [inline website/src/components/misc/winking-logo.tsx:10]: Is not it possible to do the animation in within the SVG file?

File: website/gatsby-config.js [hunk 3]
```diff
@@ -65,7 +65,7 @@ module.exports = {
       resolve: `gatsby-plugin-react-svg`,
       options: {
         rule: {
-          include: /images\/.*\.svg/,
+          include: /images/,
```
Comment (rstaib) [inline website/gatsby-config.js:68]: Why removing the more explicit filter?

File: website/src/components/misc/link.tsx (hunk 1 above)
Comment (PascalSenn) [reply website/src/components/misc/link.tsx:17]: `ref` has a quite a complex type, Its `string & React.RefObject<HTMLAnchorElement>`  I think this should work together with `React.Ref<HTMLAnchorElement>` but i really don't know. 
so this ends up being a typecast.

File: website/src/components/misc/winking-logo.tsx (hunk 2 above)
Comment (PascalSenn) [reply website/src/components/misc/winking-logo.tsx:10]: you mean with svg animations? 
I just used framer for convenience. really not fit on svg animations.

File: website/gatsby-config.js (hunk 3 above)
Comment (PascalSenn) [reply website/gatsby-config.js:68]: ah right, should i put it back in? I always have to remove it to get it working on my machine. I end up with exceptions like this: 
image

File: website/src/components/misc/link.tsx (hunk 1 above)
Comment (rstaib) [reply website/src/components/misc/link.tsx:17]: Ahhh mixed things up it's regarding the type. I will take a look again later just took a short look at it...

Comment (rstaib) [reply website/src/components/misc/link.tsx:17]: Ahhh mixed things up it's regarding the type. I will take a look again later just took a short look at it...

**Pull request closed without merging**